ansible-playbooks/roles/console/vars/main.yml

```diff
@@ -10,7 +10,7 @@ install_packages:
   - nginx
   - nmap
   - postfix
-  - postgresql-contrib
+  - "postgresql-contrib-{{ postgresql_version }}"
```

Comment:
1) Remove the double quotes, they are only required if the line starts with a {{ }} variable name (https://docs.ansible.com/ansible/latest/user_guide/playbooks_variables.html#when-to-quote-variables-a-yaml-gotcha)

2) Can you remove the "contrib", looks like it isn't required anymore.

![image](https://user-images.githubusercontent.com/27816854/96925391-b5492480-1479-11eb-82b2-3a7b02b08a4f.png)

Final line should look like:

```
  - postgresql-{{ postgresql_version }}
```

Reply:
Thanks, added your recommendations.